Src/DFC.App.MatchSkills.Services.ServiceTaxonomy/AutoMapperConfiguration.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using AutoMapper;
+using DFC.App.MatchSkills.Services.ServiceTaxonomy.Models;
+using DFC.Personalisation.Domain.Models;
+
+namespace DFC.App.MatchSkills.Services.ServiceTaxonomy
+{
+    public static class Mapping
+    {
+        private static readonly Lazy<IMapper> Lazy = new Lazy<IMapper>(() =>
+        {
+            var config = new MapperConfiguration(cfg => {
+                cfg.ShouldMapProperty = p => p.GetMethod.IsPublic || p.GetMethod.IsAssembly;
+                cfg.AddProfile<MappingProfile>();
+            });
+            var mapper = config.CreateMapper();
+            return mapper;
+        });
+
+        public static IMapper Mapper => Lazy.Value;
+    }
+
+    public class MappingProfile : Profile
+    {
+        public MappingProfile()
+        {
+            CreateMap<STSkill, Skill>()
+                .ForMember(dest => dest.Id, opt => opt.MapFrom(src => src.Uri))
+                .ForMember(dest => dest.Name, opt => opt.MapFrom(src => src.Skill))
+                .ForMember(dest => dest.AlternativeNames, opt => opt.MapFrom(src => src.AlternativeLabels))                     
+                .ConstructUsing(dest => new Skill(dest.Uri, dest.Skill, (SkillType)Enum.Parse(typeof(SkillType),dest.SkillType,true)))
+                //.ConstructUsing(dest => new Skill(dest.Uri, dest.Skill, (SkillType)Enum.Parse(typeof(SkillType),dest.SkillType,true),dest.AlternativeLabels))
```

Comment:
Let's not commit commented-out code. The code is either needed or it isn't.

Reply:
Sure, fixed.

Reply:
That also reminds me I need to do a little more work on mapping of Alternate labels. I'll refactor that later today.